index.html

```diff
@@ -82,8 +85,13 @@
     <section id="sotd">
       <p>
         This document is <strong>work in progress</strong> and is subject to
-        change. The API is being developed based on requirements from the TV
-        Control Working Group, and previously the TV Control Community Group.
+        change. The API has significantly evolved since publication as a First
+        Public Working Draft on
+        <a href="https://www.w3.org/TR/2016/WD-tvcontrol-api-20160915/"
+        title="First Public Working Draft of the TV Control API">15
+        September 2016</a>. The <code>TVTuner</code> class has been removed
+        in particular, and the API follows a more flexible source-centric
+        design.
```

Comment:
Should we mention here that our design is based on the capabilities and constraints model from https://www.w3.org/TR/mediacapture-streams?